css/index.css

```diff
@@ -63,35 +63,50 @@ em {
   margin-top: 25px;
 }
 
-form input {
+
+.contact input {
   font-size: 14pt;
   font-family: "Open Sans";
-  padding: 4px;
+  padding: 8px 50px 8px 8px;
   border: 0;
   border-bottom: 2px solid rgba(33, 33, 33, 0.2);
   margin: 0 0px 15px 0px;
-  width: 100%;
+  width: -webkit-calc(100% - 120px);
+  width: calc(100% - 120px);
 }
 
-form input:focus {
+.contact .inputButton {
+    margin-left: 10px;
+    font-size: 14pt;
+    padding: 8px;
+    font-family: "Open Sans";
+    width: 50px;
+    background: #3498db;
```

Comment:
Maybe play with lightening this up or swapping the `a` and `a:hover` colors from the landing page.